src/dist/replication/lib/log_block.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#include "log_block.h"
+
+namespace dsn {
+namespace replication {
+
+log_block::log_block() { init(); }
+
+void log_block::init()
```

Comment:
why add a function called `init` here, rather than write those code in construction function?

Reply:
I will write another constructor (in next PR) that's gonna reuse this function. So I intentionally leave it here.